src/Main.java

```diff
@@ -1,5 +1,110 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        System.out.println("Задание 1");
+        byte cat = 5;
+        System.out.println(cat);
+        short dog = 300;
+        System.out.println(dog);
+        int pig = 60000;
+        System.out.println(pig);
+        long starFish = - 1234567891;
+        System.out.println(starFish);
+        float squid = 5.3f;
+        System.out.println(squid);
+        double friedFish = 1.8f;
+        System.out.println(friedFish);
+
+        System.out.println("Задание 2");
+        double a = 27.12f;
+        System.out.println(a);
+        long b = 987678965549L;
+        System.out.println(b);
+        double c = 2.786f;
+        System.out.println(c);
+        short d = 569;
+        System.out.println(d);
+        short e = - 159;
+        System.out.println(e);
+        int f = 27897;
+        System.out.println(f);
+        byte g = 67;
+        System.out.println(g);
+
+        System.out.println("Задание 3");
+        byte LyudmilaPavlovna = 23;
+        byte AnnaSergeevna = 27;
+        byte EkaterinaAndreevna = 30;
```

Comment:
```suggestion
        byte lyudmilaPavlovna = 23;
        byte annaSergeevna = 27;
        byte ekaterinaAndreevna = 30;
```
названия переменных надо писать с маленькой буквы всегда

Reply:
также общий комментарий

нет никакого смысла в использовании byte / short вместо int, это мнимая "экономия" памяти, которая на самом деле ничего не экономит, а только доставляет хлопот в коде в виде кастов (явных приведений типов). Поэтому для целых всегда лучше использовать int по умолчанию, для вещественных - double. Экономить можно, но чуть попозже узнаешь, как. Если кратко, то когда таких переменных много, то экономия на каждой внесёт значительный вклад в общую память, но сейчас не такой случай